baseapp/baseapp_reports/models.py

```diff
@@ -3,40 +3,74 @@
 from django.contrib.contenttypes.fields import GenericForeignKey, GenericRelation
 from django.contrib.contenttypes.models import ContentType
 from django.db import models
-from django.utils.translation import gettext_lazy as _
 from model_utils.models import TimeStampedModel
 
 from baseapp_core.graphql import RelayModel
 
 
 def default_reports_count():
-    ReportModel = swapper.load_model("baseapp_reports", "Report")
+    return {"total": 0}
+
 
-    d = {"total": 0}
+def default_reports_count_full():
+    d = default_reports_count()
 
-    for report_type in ReportModel.ReportTypes:
+    ReportTypeModel = swapper.load_model("baseapp_reports", "ReportType")
+
+    for report_type in ReportTypeModel.objects.all():
         d[report_type.name] = 0
```

Comment:
In the `create_object_type_from_model` you used `name.lower()` and here it is only `name`, we need to make this consistent, also  I think we should make a unique code for report_types to be used here instead of name, because name can have white spaces and any other characters. 

Can we add a test case covering this?

Reply:
I changed the implementation, as the previous one was not working when building. Would you still like me to address this in any form?

Reply:
@luciobj yes, there is still a bug in there, you can have a name that breaks how you access it in javascript. For example if you have `Intellectual property infringement` as a name, how do you access it in JS? You will need to make sure to always use `reports_count['Intellectual property infringement']` but if you have a safer name representation you could access it with `reports_count.intellectual_property_infringement`. Got it?

Reply:
That was always the idea, but I understand why 'name' could be confusing. Change it to 'key', hopefully it will be more intuitively that way